opensciencegrid/gracc-apel/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM opensciencegrid/osg-wn:3.5-el7
```

Comment:
This is odd @djw8605 is there a reason that we inherit from the WN client instead of `software-base`?

Reply:
Probably just a bad decision at the time. Feel free to update!

Reply:
@yongleyuan please update the base to look like the other Dockerfiles that you've moved over

Reply:
Just updated it in this commit: https://github.com/opensciencegrid/images/pull/12/commits/0147944596335f73369c628b1eabcf079c7a7219.